DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -183,6 +184,27 @@ public IEnumerable<CentreSummaryForSuperAdmin> GetAllCentreSummariesForSuperAdmi
             );
         }
 
+        public IEnumerable<CentreSummaryForFindYourCentre> GetAllCentreSummariesForFindCentre()
+        {
+            return connection.Query<CentreSummaryForFindYourCentre>(
+                @"SELECT c.CentreID,
+                            c.CentreName,
+                            c.RegionID,
+                            r.RegionName,
+                            c.pwTelephone AS Telephone,
+                            c.pwEmail AS Email,
+                            c.pwWebURL AS WebUrl,
+                            c.pwHours AS Hours,
+                            c.pwTrainingLocations AS TrainingLocations,
+                            c.pwTrustsCovered AS TrustsCovered,
+                            c.pwGeneralInfo AS GeneralInfo,
+                            c.kbSelfRegister AS SelfRegister
+                        FROM Centres AS c
+                        INNER JOIN Regions AS r ON r.RegionID = c.RegionID
+                        WHERE (c.Active = 1) AND (NOT (c.Lat IS NULL) AND (NOT (c.Long IS NULL)) AND (c.ShowOnMap = 1))"

```

Comment:
Can you change this where clause so that we don't have all the unnecessary brackets?